### PR TITLE
Heading size fixes, logo fixes and other design feedback

### PR DESCRIPTION
**Pull request by helenb**
[Link to Ticket](https://torchbox.monday.com/boards/1192293412/pulses/1385296860)

### Description of Changes Made

When https://github.com/torchbox/torchbox.com/pull/84/files was merged, it inadvertently affected more than the headings in rich texts, but affected wherever we were using these font-styles elsewhere. This MR updates all those cases, as well as adding a few other tweaks noted by Ben in the figma file to the featured case study.

This PR also re-instates the changes from https://github.com/torchbox/torchbox.com/pull/77/files, which got lost when another branch changed the same lines of code.

Ben wants to have some colour on the large heading 2 drop-caps, but not the animation, so this also adds that.

### How to Test

A general check through the pattern library or individual components based on the files changed if you have time, otherwise mostly checking the code as it will take a while.

### Screenshots

<details>
  <summary>Expand to see more</summary>
Excluding screenshots as there are a lot of changes.
</details>

### MR Checklist

- [x] Add a description of your pull request and instructions for the reviewer to verify your work.
- [x] If your pull request is for a specific ticket, link to it in the description.
- [x] Stay on point and keep it small so the merge request can be easily reviewed.
- [x] Tests and linting passes.

#### Unit tests

- [ ] Added
- [x] Not required

#### Documentation

- [ ] Updated build docs
- [ ] Updated editor guidelines (https://docs.google.com/document/d/1PAWccdQ4tfaZsrEWmpDhvP3GH5RRmBOARFVp4b-kje8/edit?usp=sharing)
- [x] Not required

#### Browser testing

- [x] I have tested in the following browsers and environments (edit the list as required)
  - Latest version of Chrome on mac
- [ ] Not required

#### Data protection

- [x] Not relevant
- [ ] This adds new sources of PII and documents it and modifies Birdbath processors accordingly

#### Accessibility

- [ ] Automated WCAG 2.1 tests pass
- [ ] Manual WCAG 2.1 tests completed
- [ ] I have tested in a screen reader
- [ ] I have tested in high-contrast mode
- [ ] Any animations removed for prefers-reduced-motion
- [x] Not required

#### Sustainability

- [ ] Images are optimised and lazy-loading used where appropriate
- [ ] SVGs have been optimised
- [ ] Perfomance and transfer of data considered
- [ ] If JavaScript is needed alternatives have been considered
- [x] Not required

#### Pattern library

- [x] The pattern library component for this template displays correctly, and does not break parent templates
- [x] The styleguide is updated if relevant
- [ ] Changes are not relevant the pattern library

### CODE REVIEW

File: tbx/static_src/sass/components/_motif-heading.scss [hunk 1]
```diff
@@ -19,6 +19,10 @@
         @include reduced-motion() {
             animation: none;
         }
+
+        #{$root}--static & {
+            animation: none;
+        }
```
Comment (chris-lawton) [inline tbx/static_src/sass/components/_motif-heading.scss:25]: In hindsight I probably should have set the animation to a class of it's own so it doesn't need to be undone

File: tbx/static_src/sass/components/_pagination.scss [hunk 2]
```diff
@@ -14,12 +14,12 @@
...
 
     &__page {
-        @include font-size('heading-six');
+        @include font-size('heading-five');
         color: var(--color--heading);
         display: block;
 
         @include media-query(large) {
-            @include font-size('heading-five');
+            @include font-size('heading-four');
```
Comment (chris-lawton) [inline tbx/static_src/sass/components/_pagination.scss:22]: It's on the verbose side, but you could use `font-size: nth(map-get($large-font-sizes, 'heading-four'), 1);` here to access the font size (and would need the same for line-height) so not to include the CSS for mobile for 'heading-four' when it isn't used.

image

**Review by chris-lawton — APPROVED** [review]

File: tbx/static_src/sass/components/_motif-heading.scss (hunk 1 above)
Comment (helenb) [reply tbx/static_src/sass/components/_motif-heading.scss:25]: I think I'll leave it as is for now - if we find we're overriding it all over the place we can refactor

File: tbx/static_src/sass/components/_pagination.scss (hunk 2 above)
Comment (helenb) [reply tbx/static_src/sass/components/_pagination.scss]: Updated in 6088e413a9b8d7790712ee88330694f885824372

**Pull request merged**